examples/FirebaseDemo_ESP8266/FirebaseDemo_ESP8266.ino

```diff
@@ -20,11 +20,19 @@
 #include <ESP8266WiFi.h>
 #include <FirebaseArduino.h>
 
-void setup() {
-  Serial.begin(9600);
+// Set these to run example.
+#define FIREBASE_HOST "example.firebaseio.com"
+#define FIREBASE_AUTH "token_or_secret"
+#define WIFI_SSID "SSID"
+#define WIFI_PASSWORD "PASSWORD"
+
+void ConnectWifi(const String& ssid, const String& password = "") {
```

Comment:
That get tricky if the header is part of the sample and not of the lib, because when you save the project in it's own directory it won't source what's included from `../../`